building.html

```diff
@@ -0,0 +1,193 @@
+
+      <h2>Building</h2>
+
+      <p>Currently, the complete game client can only be built for Windows, using Visual Studio.</p>
+      <p>By default these steps will compile a client targeting your machine architecture. It is possible to build a 64-bit game client, but only the 32-bit (x86) client is officially supported.</p>
+
+
+      <h3 id="dependencies">Library Dependencies</h3>
+
+      <p>Plasma requires a number of third-party libraries as dependencies. All required libraries are available through the <a href="https://github.com/microsoft/vcpkg">vcpkg package manager</a> and can be downloaded as part of the build process, or can be built using their individual build instructions.</p>
+
+      <details>
+        <summary>Expand to view the complete list of libraries</summary>
+
+        <p>Plasma currently requires the following third-party libraries:</p>
+        <ul>
+          <li><a href="https://www.python.org/downloads/">Python 3.8 or higher</a></li>
+          <li><a href="https://www.xiph.org/downloads/">libOgg and libVorbis</a></li>
+          <li><a href="http://www.slproweb.com/products/Win32OpenSSL.html">OpenSSL</a></li>
+          <li><a href="https://openal-soft.org/">OpenAL Soft</a></li>
+          <li><a href="https://libexpat.github.io/">Expat</a></li>
+          <li><a href="https://libjpeg-turbo.org/">libjpeg</a></li>
+          <li><a href="http://www.libpng.org/">libpng</a></li>
+          <li><a href="https://zlib.net/">zlib</a></li>
+          <li><a href="https://curl.se/">libcurl</a></li>
+          <li><a href="https://github.com/zrax/string_theory/">string_theory</a></li>
+          <li><a href="https://think-async.com/Asio/">Standalone ASIO</a></li>
+        </ul>
+
+        <p>The following libraries are optional:</p>
+        <ul>
+          <li><a href="https://cairosvg.org/">CairoSVG</a> (for building resource.dat)</li>
+          <li><a href="https://python-pillow.org/">Pillow</a> (for building resource.dat)</li>
+          <li><a href="https://www.freetype.org/">FreeType</a> (for plFontConverter)</li>
+          <li><a href="https://www.qt.io/download-open-source">Qt5</a> (for the GUI tools)</li>
+          <li><a href="https://www.webmproject.org/">VPX and WebM</a> (for video)</li>
+          <li><a href="https://www.opus-codec.org/">Opus</a> (for video and voice chat)</li>
+          <li><a href="https://www.speex.org/downloads/">Speex</a> (for legacy voice chat)</li>
+          <li>Autodesk 3ds Max 7 (non-free SDK; for 3ds Max plugin)</li>
+        </ul>
+      </details>
+
+
+      <h3 id="building-windows">Building on Windows</h3>
+
+      <p>Compiling on Windows requires Visual Studio and the Windows SDK. We recommend <a href="https://visualstudio.microsoft.com/downloads/">Visual Studio 2019 Community Edition</a>, which is available as a free download. During the installation of Visual Studio, the Windows SDK can be installed by selecting the “Desktop development with C++” workload.</p>
```

Comment:
This is a little wordy and overly specific IMO. The goal is to get a working Windows C++ compiler, so I think it should just say 'install the "desktop development with C++" workload". I'm not certain the wording in the suggestion is perfect, though.

```suggestion
      <p>Compiling on Windows requires Visual Studio. We recommend <a href="https://visualstudio.microsoft.com/downloads/">Visual Studio 2019 Community Edition</a>, which is available as a free download. Install the “Desktop development with C++” workload.</p>
```